Ansible/roles/cloudstack-manager/tasks/ubuntu.yml

```diff
@@ -85,6 +77,14 @@
 
 - include: ./setupdb.yml
 
+- name: Ensure CloudStack packages are installed (now that mgmt server "db.properties" and "key" files are present)
```

Comment:
This is 'fine' as long as it is in line with the documented order of steps.

Reply:
yes, that's how it's supposed to work, based on the content of the debian.post-install script - if "db.properties" exists in the mgmt folder, the symlink is created. The identical story for the "key" file.